app/libs/angular-storage.js

```diff
@@ -0,0 +1,147 @@
+(function(angular, factory) {
```

Comment:
The formatting does not match the rest of the files in the project. You should provide a License if this is Open Source code or if this is your code, then format it accordingly.